src/main/java/by/onliner/recorder/VideoManager.java

```diff
@@ -0,0 +1,51 @@
+package by.onliner.recorder;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+public class VideoManager {
+
+    private static final ThreadLocal<VideoRecorder> threadRecorder = new ThreadLocal<>();
+    private static final List<VideoRecorder> recorders = Collections.synchronizedList(new LinkedList<>());
```

Comment:
Are you sure you need both threadRecorder and recorders? If yes, why?

Reply:
In this situation i think i don't need both threadRecorder and recorders. I need to use only threadRecorder